index.html

```diff
@@ -368,20 +388,63 @@ <h3>
           <p class="note" title="HTTP API">
 There is no HTTP API specified for the update process leaving programmatic
 registrations and management to be defined by each implementation.
-          </p>
+</p>
         </section>
 
         <section>
           <h3>
 Deactivate (Revoke)
           </h3>
           <p>
-To delete the DID document, the <code>did.json</code> has to be removed or has
-to be no longer publicly available due to any other means.
+To delete the DID document, the associated file or files have to be removed or
+made no longer publicly available due to any other means.
           </p>
         </section>
       </section>
 
+      <section>
+      <h2>
+Webserver Configuration
+      </h2>
+      <h3>
+MIME Types
+      </h3>
+      <p>
+Webservers MUST be configured to associate proper content types with registered
+filename extensions.
+      <pre class="note nohighlight" title="MIME Types">
```

Comment:
I think that instead of this table (listing just `application/json`), we should refer readers to the [IANA Consideration section](https://w3c.github.io/did-core/#iana-considerations) of the DID spec, for content type to extension mapping.

Reply:
I made that change. I did not see a good way to link to an html anchor within a specification, so I just added an html link.

db763a18f0242fdbc9894fb28e98c9fd9e180a3f